src/ray/object_manager/plasma/client.h

```diff
@@ -229,6 +229,11 @@ class PlasmaClientInterface {
   /// \param object_ids The list of IDs of the objects to delete.
   /// \return The return status. If all the objects are non-existent, return OK.
   virtual Status Delete(const std::vector<ObjectID> &object_ids) = 0;
+
+  /// Get the current debug string from the plasma store server.
+  ///
+  /// \return the debug string if successful, otherwise return an error status.
+  virtual StatusOr<std::string> GetMemoryUsage() { return std::string(); }
```

Comment:
should be = 0 so it stays a pure interface
```suggestion
  virtual StatusOr<std::string> GetMemoryUsage() = 0;
```